### ISSUE TITLE
Document ILM settings 

### ISSUE THREAD

**Issue by eddieturizo** [elastic/docs#878]
`index.lifecycle.poll_interval` appears to have a default value of `10m` but it does not seem to be explicitly [documented](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/using-policies-rollover.html). 

We should document ILM settings like this and perhaps others that are currently not documented.

**Comment by kpstolk** [comment]
@eddieturizo Is `index` a typo or is that setting also supported?
Because the documentation only mentions indi**ces**.lifecycle.poll_interval.

**Comment by eddieturizo** [comment]
@kpstolk Good catch. Indeed it should be `indices` instead of `index`. Thank you.

**Comment by dedemorton** [comment]
@eddieturizo This repo is for issues related to the tooling we use to generate the documentation. Can you please submit your request to the product repo instead: 

https://github.com/elastic/elasticsearch/issues/new

I just happened to notice this while I was looking for something else....

**Comment by dedemorton** [comment]
Please close this after you've created the issue in the elasticsearch repo.
